### PR TITLE
Patch 3 - Audio Device Change Event & Mute/Deafen State Fix

### PR DESCRIPTION
**Pull request by gitagogaming**
Fixed muteState when deafening - Previous patch mistakenly forgot to add muteState=0 when toggling...  
New audio device change event to make sure your device is always set properly.
image

### CODE REVIEW

File: src/handlers/discord/voiceStateHandler.js [hunk 1]
```diff
@@ -149,9 +154,8 @@ class VoiceStateHandler {
       this.TPClient.settingUpdate("Plugin Connected", "Disconnected");
       this.TPClient.stateUpdate("discord_connected", "Disconnected");
       this.DG.connected = false;
-      if (platform != "win32") {
-        // return this.Discord.doLogin();
-        print("wooopsie.. this is not windows.., gitago forgot to fix the reglogin he didnt think was needed");
+      if (platform != "win32" && this.DG.pluginSettings["Skip Process Watcher"] === "No") {
```
Comment (spdermn02) [inline src/handlers/discord/voiceStateHandler.js:157]: This should be `||` and the value in the right should be "Yes" (probably want to do case insensitive check as well)

**Review by spdermn02 — COMMENTED** [review]
Check out my comment

File: src/handlers/discord/voiceStateHandler.js (hunk 1 above)
Comment (gitagogaming) [reply src/handlers/discord/voiceStateHandler.js]: Got it corrected, Thanks

**Review by spdermn02 — DISMISSED** [review]

**Comment by gitagogaming** [comment]
My mistake, I forgot to do a version update

**Review by spdermn02 — APPROVED** [review]

**Pull request merged**